package-lock.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "web-world-wind",
-  "version": "0.0.1",
+  "name": "@nasaworldwind/webww",
+  "version": "0.1.2",
```

Comment:
The new name corresponds to how the package will be identified in npm's repository, including its scoping within our org.

The version number was wrong with respect of what's in package.json, It's corrected now. 

I never manually changed package-lock.json, this was caused by ['npm version'](https://docs.npmjs.com/cli/version). When I found out that I didn't need to publish to npm to test the package (and hence, I needed to update the version number each time I published), I started to manually change the version in package.json, since 'npm version' commits to Git every time is executed and I didn't want to add noise to my commit history. I didn't review package-lock.json, however (the package works even with this discrepancy). Thanks for catching that.

Note that I left the nonsensical "0.0.1" version that is meant to be changed when we're ready to publish.